JavaScript/CodingConventions.md

```diff
@@ -180,32 +180,32 @@ When using `switch` statements:
  - Use `break`, `return` or `throw` for each case other than `default`
  - The `default` case should always be last.
 
-As with all our whitespace guidelines, switch statements follow JSLint.
+Use 1 tab indentation for `case` / `default` statements.
 
 ```javascript
 // bad
 switch (foo) {
-    default:
-        z();
-    case 'bar':
-    case 'foobar':
-        x();
-        break;
-    case 'baz':
-        y();
-}
-
-// good
-switch (foo) {
 case 'bar':
 case 'foobar':
-	x();
-	break;
+    x();
```

Comment:
As described in above article it can "unhealthy" topic. I was just curious that's all. It would be helpful to have one standard since I would probably forget it and had to revert my changes. Hopefully I'm not working a lot with JS lately so no worries. 
It is OK to be consistent within given Team/Language so your above comment is absolutely valid.